assignments/arrays.js

```diff
@@ -63,34 +63,52 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log('Car 33 is a ' + inventory[32].car_year + ' ' + +inventory[32].car_make + ' ' + inventory[32].car_model);
```

Comment:
When I run this code, I get `Car 33 is a 2011 NaN Wrangler.` It looks like the second `+` before `inventory[32].car_make` is what's causing that -- when I remove it, I get the right solution.
What you have generally works as a way to piece together a string, but I'd typically recommend using a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead.